src/gt4py/next/iterator/transforms/inline_lifts.py

```diff
@@ -14,14 +14,17 @@
 
 import dataclasses
 import enum
+import functools
+import operator
 from collections.abc import Callable
-from typing import Optional
+from typing import Optional, TypeGuard
 
 import gt4py.eve as eve
 from gt4py.eve import NodeTranslator, traits
 from gt4py.next.iterator import ir
 from gt4py.next.iterator.ir_utils import ir_makers as im
 from gt4py.next.iterator.transforms.inline_lambdas import inline_lambda
+from gt4py.next.iterator.transforms.trace_shifts import TraceShifts, copy_recorded_shifts
```

Comment:
Good example for a case where I really dislike this because it does not increase readabilty at all:
```
trace_shifts.TraceShifts.apply(...)
trace_shifts.copy_recorded_shifts(...)
```